.gitignore

```diff
@@ -5,3 +5,4 @@ website/build
 coverage/
 node_modules
 .docusaurus
+.idea
```

Comment:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
thanks for the feedback @pokonski , reverted the changes in that file.